deploy/stacks/param_store_stack.py

```diff
@@ -92,6 +93,13 @@ def __init__(
             parameter_name=f'/dataall/{envname}/pivotRole/enablePivotRoleAutoCreate',
             string_value=str(enable_pivot_role_auto_create),
         )
+        
+        aws_ssm.StringParameter(
+            self,
+            f'dataallAllowedOrigins{envname}',
+            parameter_name=f'/dataall/{envname}/allowedOrigins',
+            string_value=str(allowed_origins),
+        )
```

Comment:
do we need this SSM Parameter if just using env variable in Lambdas?

Reply:
Added this incase we want to use for local deployment testing. If this is not required for local deployment then i will remove this Noah.

Reply:
for local testing allowed origins should just default to `*` in my opinion - think we can remove this SSM param

Reply:
Removed this Noah